api/v1beta1/verticadb_types.go

```diff
@@ -39,10 +39,12 @@ const VerticaDBAPIVersion = "vertica.com/v1beta1"
 type VerticaDBSpec struct {
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default:=IfNotPresent
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
```

Comment:
It is probably out of scope in this PR, but are you going to fill in the x-descriptors?  You can indicate if a field is expecting a Secret or ConfigMap (see here https://github.com/operator-framework/operator-lifecycle-manager/blob/e0eea22/doc/design/building-your-csv.md)

Reply:
Yes I was focused on mandatory fields but I guess on the next PR I can add x-descriptors too.